unison-src/transcripts/add-run.output.md

```diff
@@ -290,19 +290,21 @@ main = '5
 
 ```
 ```ucm
-.> run main
+scratch/main> run main
 
   5
 
-.> add.run .an.absolute.name
+scratch/main> add.run .an.absolute.name
 
   ⍟ I've added these definitions:
   
     .an.absolute.name : Nat
 
-.> view .an.absolute.name
+scratch/main> view .an.absolute.name
 
-  .an.absolute.name : Nat
-  .an.absolute.name = 5
+  .__projects._ed77def0_0f5b_4a68_9563_0c705b131e21.branches._5a2c3951_0700_4331_8455_a27805a78722.an.absolute.name :
+    Nat
+  .__projects._ed77def0_0f5b_4a68_9563_0c705b131e21.branches._5a2c3951_0700_4331_8455_a27805a78722.an.absolute.name =
+    5
```

Comment:
I'm a little  surprised  by this behaviour tbh; I didn't think absolute paths were considered relative to the project root already 🤔